README.md

```diff
@@ -85,15 +85,28 @@ Open a new bash shell
 # test the application
 
 # test using httpie (installed automatically in Codespaces)
-http localhost:4120/version
+http localhost:8080/version
 
 # test using curl
-curl localhost:4120/version
+curl localhost:8080/version
+
+# test using lr
+docker run --rm --network=host ghcr.io/retaildevcrews/ngsa-lr:beta -s http://localhost:8080 --max-errors 1 -f baseline.json
 
 ```
 
 Stop ngsa by typing Ctrl-C or the stop button if run via F5
 
+### Run Application Unit Tests
+
+To run unit tests
+
+```bash
+
+mvn clean package -Dmaven.test.skip=false
```

Comment:
If you only want to run the unit tests you can also run `mvn test` - no need for building the package